builder/builder.go

```diff
@@ -47,8 +48,10 @@ var UseNewSwupdServer = false
 type Builder struct {
 	Buildscript string
 	Buildconf   string
+	Mixerdir    string
 
 	Bundledir   string
+	Lbundledir  string
```

Comment:
Optional: LocalBundleDir, MixerDir. (We'll eventually fix the other names in this struct)